src/bresult.mli

```diff
@@ -9,198 +9,174 @@
 
 type 'a sequence = ('a -> unit) -> unit
 type 'a equal = 'a -> 'a -> bool
-type 'a ord = 'a -> 'a -> int
-type 'a printer = Format.formatter -> 'a -> unit
-
-(** {2 Basics} *)
+type 'a ord = 'a -> 'a -> Comparison.comparison
```

Comment:
This might actually fit better in the `Bord ` module. But I'll have to look into what's there and how it's used first.